### PR TITLE
TunerStudio command x14 cleanup

### PR DESCRIPTION
**Pull request by nmschulte**
- split the TS X14 "sub" command from the high-level TS command enum
- remove unused TPS calibration commands and implementation
- cleanup TS template symbol names
- ~~tidy indent/`IFDEF` macros in DC motors~~
- ~~use ADC conversion function in ETB~~
----
as far as I can tell, this should result in no breaking changes for FOME: no commands changed value/ID, and no releases/bundles exposed the `cmd_calibrate_tps*` symbols.

### CODE REVIEW

File: firmware/controllers/actuators/dc_motors.cpp [hunk 1]
```diff
@@ -11,76 +11,76 @@
...
-				pinEnable,
-				&m_pinEnable,
-				clampedFrequency,
-				0
-			);
-#endif // EFI_UNIT_TEST
-
-			dcMotor.configure(m_pwm1, wrappedDir1, wrappedDir2, isInverted);
-		}
+void DcHardware::start(bool useTwoWires,
```
Comment (nmschulte) [inline firmware/controllers/actuators/dc_motors.cpp:14]: this is just an indentation cleanup

File: firmware/controllers/algo/engine_types.h [hunk 2]
```diff
@@ -502,27 +502,26 @@ enum class trigger_type_e : uint32_t {
...
-	TS_RESET_TLE8888 = 8,
-	TS_COMMAND_9 = 9,
-	TS_WRITE_FLASH = 10,
-	TS_COMMAND_11 = 11,
-	TS_COMMAND_12 = 12,
-	TS_COMMAND_13 = 13,
-	TS_COMMAND_14 = 14,
-	TS_COMMAND_15 = 15,
-	TS_COMMAND_16 = 16,
+	TS_UNUSED_1 = 1,
```
Comment (nmschulte) [inline firmware/controllers/algo/engine_types.h:505]: :thinking:

File: firmware/controllers/algo/engine_types.h [hunk 3]
```diff
@@ -502,27 +502,26 @@ enum class trigger_type_e : uint32_t {
...
+	TS_UNUSED_11 = 11,
+	TS_UNUSED_12 = 12,
+	TS_UNUSED_13 = 13,
+	TS_UNUSED_14 = 14,
+	TS_UNUSED_15 = 15,
+	TS_UNUSED_16 = 16,
 	TS_CLEAR_WARNINGS = 17,
 	TS_IGNITION_CATEGORY = 18,
 	TS_INJECTOR_CATEGORY = 19,
 	TS_X14 = 20,
```
Comment (nmschulte) [inline firmware/controllers/algo/engine_types.h:524]: Why is 0x14 "subbed"?

File: firmware/controllers/actuators/dc_motors.cpp (hunk 1 above)
Comment (mck1117) [reply firmware/controllers/actuators/dc_motors.cpp:14]: unrelated changes in an unrelated PR, sigh

Comment (nmschulte) [reply firmware/controllers/actuators/dc_motors.cpp:14]: I'll drop the last two commits and put them in separate PRs, no worries.

**Pull request merged**